src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -669,6 +669,38 @@ describe('Testing Edit concept actions ', () => {
     });
   });
 
+  it('should unretire a Mapping', () => {
```

Comment:
How is this related to the ticket?

Reply:
It was affecting my coverage but I have now created a separate [PR](https://github.com/openmrs/openmrs-ocl-client/pull/382) for it.